src/kcc3/data_types.rs

```diff
@@ -38,6 +38,7 @@ impl Player {
         }
     }
 
+    #[allow(dead_code)]
```

Comment:
This function seems non-trivial so I kept it around despite it being unused -- don't know if we should keep it around.

Reply:
Hmm, there is a slight chance we'll use it in the future, but no idea. I guess we can as well remove it for now, it's not complicated either.

Reply:
good thing we have VCSes then :xD: I'll drop it